go.sum

```diff
@@ -0,0 +1,67 @@
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Should this file be ignored?

Reply:
I believe so, yes. https://www.reddit.com/r/golang/comments/97dxj1/should_i_add_the_gosum_file_to_the_repo_of_a/